targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT_PSA/prebuilt/export_binaries.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2017-2018 ARM Limited
```

Comment:
Not needed anymore, targets.json defines a delivery_dir

Reply:
The file removed